student-store-ui/src/components/Sidebar/SidebarActive/SidebarActive.jsx

```diff
@@ -0,0 +1,344 @@
+import * as React from "react";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import Cart from "../../Cart/Cart";
+
+export default function SidebarActive({
+  cartItems,
+  products,
+  itemsOnCart,
+  setItemsOnCart,
+}) {
+  useEffect(() => {
+    setItemsOnCart(Object.keys(cartItems)?.length > 0);
+  }, [cartItems]);
+
+  Object.values(cartItems).every((e) => e === 0) ? (
+    setItemsOnCart(false)
+  ) : (
+    <></>
+  );
+
+  const [total, setTotal] = useState("");
+  const [subtotal, setSubtotal] = useState("");
+  const [taxes, setTaxes] = useState("");
+  const [formStudentName, setFormStudentName] = useState({
+    studentName: "",
+  });
+  const [formStudentEmail, setFormStudentEmail] = useState({
+    studentEmail: "",
+  });
```

Comment:
Don't need to have two separate states representing each input, see the React forms demo that [Paige sent](https://github.com/paigegodfrey/react-forms-demo/blob/main/src/components/ListItemForm/ListItemForm.jsx) and the lab solutions